pjrt-plugin/iree/integrations/pjrt/common/compiler.h

```diff
@@ -26,6 +27,11 @@ class CompilerJob {
  public:
   virtual ~CompilerJob() = default;
 
+  // Enables crash dumping via an ArtifactDumper transation. The transaction
```

Comment:
transaction

Reply:
I wonder if the 'c' key is going out on my keyboard... all of the times I've misspelled that.